simtools/visualization/visualize.py

```diff
@@ -531,3 +542,137 @@ def plot_hist_2D(data, **kwargs):
     fig.tight_layout()
 
     return fig
+
+
+@u.quantity_input(x=u.m, y=u.m, radius=u.m)
+def get_telescope_patch(name, x, y, radius):
+    """
+    Collect the patch of one telescope to be plotted by plot_array.
+
+    Parameters
+    ----------
+    name: str
+        Name of the telescope (type).
+    x: astropy.units.m
+        x position of the telescope in meters.
```

Comment:
Generally: is there a strong reason to ask for meters, and not for units of lengths? If you anyway convert to value, you can also convert to meters first.

Reply:
hmm yes, because we want to plot it in meters. This function receives the positions from  "pos_x" and "pos_y" keywords, which we define in meters. I dont see a reason to expand the function here. Do you think it is necessary?

Reply:
All what you need would be to use instead of `x` the converted value `x.to(u.m)`, so that is not a large expansion.